web/src/scheduled_messages.js

```diff
@@ -18,7 +18,7 @@ import * as timerender from "./timerender";
 export const MINIMUM_SCHEDULED_MESSAGE_DELAY_SECONDS = 5 * 60;
 export const SCHEDULING_MODAL_UPDATE_INTERVAL_IN_MILLISECONDS = 60 * 1000;
 
-export let scheduled_messages_data = [];
+export let scheduled_messages_data = {};
 
```

Comment:
Can you add a comment regarding what is the key and value of the dictionary here?

Reply:
Sure, will do that